src/protocol-only/notebookConcatDocument.ts

```diff
@@ -2,8 +2,8 @@
 // Licensed under the MIT License.
 
 'use strict';
-import * as vscode from 'vscode';
-import * as protocol from 'vscode-languageclient/node';
+import * as vscodeUri from 'vscode-uri';
```

Comment:
@Heejaechang @bschnurr this file should now be not using any of the vscode types.